UPGRADING.md

```diff
@@ -21,7 +21,7 @@ The data node configuration file can be found in `$DATANODE_HOME/config/data-nod
 
 ### Settings added in 0.71.4
 
-**_Rate Limiting_** - In the past the datanode API did rate limiting only by the number of subscriptions open per IPs. With the new version a rate limit per IP was introduced. The server now allows 20 API calls per second on average with a burst of up to 100 api calls in a short period of time.
+**_Rate Limiting_** - In the past the datanode API did rate limiting only by the number of subscriptions open per IPs. With the new version a rate limit per IP on the unary calls for the RESY, GraphQL and grpc APIs was introduced. The server now allows 20 API calls per second on average with a burst of up to 100 api calls in a short period of time.
```

Comment:
on the unary calls for the RESY, - should be REST

Reply:
thanks. addressed.